composer.json

```diff
@@ -6,12 +6,14 @@
     },
 
     "require": {
-        "slim/slim": "^3.0"
+        "slim/slim": "^3.0",
+        "predis/predis": "^1.1"
     },
 
     "require-dev": {
         "phpspec/phpspec": "^3.0",
         "squizlabs/php_codesniffer": "2.7.*",
-        "behat/behat": "^3.3"
+        "behat/behat": "^3.3",
+        "phpunit/phpunit": "^6.1"
```

Comment:
Doesn't look like you are using PHPUnit yet - I'd remove it for now. (You can use it if you want, although you could get by just fine with just PHP spec for this project)